### ISSUE TITLE
Update hazard data window - layer name

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/thinkhazard#822]
Where hazard set name is shown, use layer title instead.
@ErikKBethke testing potential name lengths

Camptocamp:
On basis of above, to advise possible solutions to displaythe lengths we have
Where descriptor is 'hazard set' on screen, use 'Displayed data'

**Comment by fjacon** [comment]
@stufraser1 could you give us an example or two ? Thanks

**Comment by stufraser1** [comment]
@ErikKBethke  Have you made progress on updating the layer title to be more user-reable?
We need to assign some in GeoNode, to provide final examples to camptocamp.

**Comment by ErikKBethke** [comment]
I have been holding off on actually updating the layer titles in GeoNode, but can start pushing some examples to the production server (a great example would be the [GEM layers](https://www.geonode-gfdrrlab.org/layers/?limit=100&offset=0&title__icontains=gem) to test with).

If you would like me to update what I have, please advise. There are quite a few layers and I am still working through the naming convention for most.

Having looked into this a bit, I believe we would run into issues with the new naming convention being used in the legend due to two issues: the layer name is quite lengthy and would likely need to be abbreviated, and using layer title instead of hazard set would not necessarily make sense (given the return periods being in layer title which is not how the legend operates). It might make sense to also update hazard set titles and continue to use those.

**Comment by ErikKBethke** [comment]
Regarding the changes to be made, I would think that once the changes go live it would be fairly simple to update the legend by modifying the way that the [report_hazard_category.jinja2](https://github.com/GFDRR/thinkhazard/blob/master/thinkhazard/templates/report_hazard_category.jinja2) is set up. Could be as simple as modifying the following code to deal with lengthy params:

`          <dt>Hazard set</dt>
          <dd>{{hazardset.id}}</dd>`

**Comment by fjacon** [comment]
@stufraser1 : I agree with @ErikKBethke that using layer title instead of hazard set would not necessarily make sense.

**Comment by stufraser1** [comment]
@fjacon I understand the hesitation to use layer title because of the return period value being in layer title. Internally we had discussed to update hazard set to be the same as layer title, without the return period value as Erik noted ("It might make sense to also update hazard set titles and continue to use those"). 
= It is agreed to keep using hazard set.

The issue remains that hazard set string would become longer as they comprise the country/region and peril spelled out in full

Examples have already been included at https://www.geonode-gfdrrlab.org/search/?title__icontains=Cyclone&limit=100&offset=0
The hazard sets (not yet updated) for the layers on this link would be
Cyclone FSM+Guam KIT (from CY_FSMGUAM_KIT)
Cyclone Vietnam KIT (from CY_VN_KIT)
Cyclone Philippines KIT (from CY_PH_KIT)

Another example https://www.geonode-gfdrrlab.org/layers/hazard:ts_mih_rp500:
hazard set 'GTM Global Tsunami Hazard'
layer titles 'Global Tsunami Hazard GTM RP500' etc

**Comment by stufraser1** [comment]
need to provide table of current name / layer ID to new name, for new SQL query to perform conversion.
